src/main/java/org/broadinstitute/hellbender/tools/spark/sv/discovery/SvDiscoveryUtils.java

```diff
@@ -154,4 +153,27 @@ public static void writeSAMRecords(final List<GATKRead> reads, final Set<String>
         samRecords.sort(localComparator);
         SVFileUtils.writeSAMFile( outputPath, samRecords.iterator(), cloneHeader, true);
     }
+
+    /**
+     * this exist because for whatever reason,
```

Comment:
Put in a TODO comment reminding us to get to the bottom of this behavior. If it's something wrong with our data, we should fix it, otherwise we should file a bug against htsjdk.